src/parseTools.js

```diff
@@ -1253,3 +1253,8 @@ function makeMalloc(source, param) {
   }
   return `abort('malloc was not included, but is needed in ${source}. Adding "_malloc" to EXPORTED_FUNCTIONS should fix that. This may be a bug in the compiler, please file an issue.');`
 }
+
+function getDynCaller(sig) {
+  return MINIMAL_RUNTIME ? `dynCalls[${sig}]` : `Module["dynCall_"+${sig}]`;
```

Comment:
Based on your explanation is sounds like we avoid the extra lookup here by doing just `dynCall_${sig}` in the MINIMAL_RUNTIME case?

Reply:
In fact will that work in both cases?   (i.e. can we avoid the Module object in the normal runtime case too?)

Reply:
This function generates the lookup for dynamic string indexing case, so we would need to index the local scope, something like

```js
function getDynCaller(sig) {
  return `(ENVIRONMENT_IS_NODE ? global : window)['dynCall_'+${sig}]`;
}
```
which would be quite tedious (and not sure if it's right in `MODULARIZE` case)

Reply:
Oh I see `sig` is dynamic.. sorry I misunderstood.